front/js/order.es6

```diff
@@ -22,35 +22,41 @@
     $(DOM.submitBtn).on('click', DOM.submit, submitOrder);
   }
 
-  const getProductId = $target => $target.closest(DOM.productRow).data('product-id');
-  const getProductCount = $target => $target.closest(DOM.productRow).find(DOM.productCount).val();
-  const getProductName = $target => $.trim($target.find('a').text());
-
-  const getProductsData = () => $(DOM.productRow).map((_, el) => {
-    const $el = $(el);
+  const getProductData = ($el) => {
+    const $row = $el.closest(DOM.productRow);
     return {
-      id: getProductId($el),
-      name: getProductName($el),
-      quantity: getProductCount($el),
+      id: parseInt($row.data('id'), 10),
+      quantity: parseInt($row.find(DOM.productCount).val(), 10),
+      name: $.trim($row.find('a').text()),
     };
-  }).get();
+  };
+
+  const getProductsData = () => $(DOM.productRow)
+    .map((_, el) => {
+      const $el = $(el);
+      let data = getProductData($el);  // Ignore ESLintBear (prefer-const)
+      data.price = parseFloat($el.closest(DOM.productRow).data('price'));
```

Comment:
maybe we should put this price to `getProductData` side?
It's up to you

Reply:
`getProductsData` and price field are used only by a purchase action, so we have to do it in that way